ripe/atlas/tools/commands/measure.py

```diff
@@ -306,8 +389,8 @@ def _get_measurement_kwargs(self):
 
         r["is_oneoff"] = True
         if self.arguments.interval or spec["times"]["interval"]:
-            r["is_oneoff"] = False
             r["interval"] = self.arguments.interval
+            r["is_oneoff"] = False
             self.arguments.no_report = True
```

Comment:
On Fri, Sep 25, 2015 at 11:16 AM, Andreas Strikos
<notifications@github.com> wrote:
>
> In ripe/atlas/tools/commands/measure.py:
>
> >              r["interval"] = self.arguments.interval
> > +            r["is_oneoff"] = False
> >              self.arguments.no_report = True
>
> what was the point of this?

Cosmetics - no functional change.

-- 
"If you want to go fast, go alone. If you want to go far, go together."